src/server/routes/tables.ts

```diff
@@ -37,7 +37,7 @@ export default async (fastify: FastifyInstance) => {
     const { data, error } = await pgMeta.tables.retrieve({ id })
     await pgMeta.end()
     if (error) {
-      request.log.error({ error, req: request.body })
+      request.log.error(JSON.stringify({ error, req: request.body }))
```

Comment:
This...fixes the response sent back to the client?

Reply:
Ostensibly yeah, `request.log.error` accepts a string, not sure what the handler does but once I pass it an object all bets are off

Reply:
Not sure where the `accept-version` stuff comes from.

Reply:
Interesting, in fact setting `req` to anything triggers the error. Doesn't happen when I set e.g. `a`. Not sure why that matters.

Reply:
Ahh makes sense now. I think fastify sets that key and that key is overriden by you here. So the internal logging module is trying to do req.headers.accept-version and that ends up throwing the error.

Reply:
`JSON.stringify`ing `request.body` seems fine. Either way simply passing a string as per docs seems to have fixed it, so probably not worth it to go further down the rabbit hole.